server/controllers/articleController.js

```diff
@@ -230,8 +226,7 @@ class ArticleController {
   */
   static async likeOrDislikeArticle(req, res) {
     const { user } = req;
-    const { liked, resourceId, type } = req.body.liked;
-    const likes = parseBool(liked);
```

Comment:
Great Job Kafilat, however, I would like to know the reason for this removal?, I do understand that you probably found a better way to implement this feature, but I'm curious. Thanks

Reply:
Thank you @encodedBicoding, the function was converting the string received from the request body to boolean before sending it to the database so I removed it since boolean can be sent directly in tne request body.